src/components/Dropdown/Dropdown.tsx

```diff
@@ -121,30 +122,54 @@ export class Dropdown extends React.PureComponent<IDropdownProps, IDropdownState
                     role="combobox"
                     style={dropdownContainerStyle}
                 >
-                    <span className={dropdownTitleClassName} title={this.props.label}>
-                        {icon && (
-                            <Icon iconName={icon}></Icon>
-                        )}
-                        {selectionTextObj.text}
-                        {this.props.displaySelection && this.props.showArrowIcon &&
-                            <Icon className={dropdownIconClassName} iconName={arrowIcon}></Icon>
-                        }
-                    </span>
+                    {isError &&
+                        < Tooltip
+                            content={this.props.validationErrorMessage}
+                            delayMs={500}
+                            directionalHint={DirectionalHint.rightCenter}
+                            className={'tooltip-error'}>
+                            <span className={dropdownTitleClassName} title={this.props.label}>
+                                {icon && (
```

Comment:
if i am not mistaken this is the same code as below. since it is a single element you can easily put it in some variable and use it in both places.

Reply:
Moved the code to variable